### ISSUE TITLE
Clarification on Loss Function in Equation (2) and `blip_cir.py`

### ISSUE THREAD

**Issue by lucas-ventura** [OmkarThawakar/composed-video-retrieval#7]
Hi! Great work on this project, and thank you for sharing your code.

I have a question regarding the loss function. In **Equation (2)** of the paper, there are three losses mentioned:  $f(q, t) + f(q, d) + f(e(d), t)$  
However, in [`blip_cir.py`](https://github.com/OmkarThawakar/composed-video-retrieval/blob/29e2e15fdd81772b57ca882e3757270c9990a590/src/model/blip_cir.py#L157C9-L158C1), I noticed the following implementation:
```python
loss = 0.8 * self.loss(query_feat, tar_img_feat, self.temp) + 0.2 * self.loss(query_feat, tar_img_mmfeat, self.temp)
```

Could you please elaborate on how this implementation corresponds to the loss function in the paper? 
Thank you for your time and clarification!

Best regards,  
Lucas
